src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -553,9 +576,13 @@ public async Task TestSettingConfigurationCreatesCorrectClasses()
             Assert.IsNotNull(configuration, "TryGetRuntimeConfiguration should set the config in the out parameter.");
             Assert.IsTrue(isConfigSet, "TryGetRuntimeConfiguration should return true when the config is set.");
 
+            ConfigurationPostParameters expectedParameters = GetCosmosConfigurationParameters();
```

Comment:
Shouldn't this be specific to which configurationEndpoint you are testing?

Reply:
It doesn't matter since this is just used to verify the config was set correctly. The config is the same with both endpoint, it's just the format of the payload that changes.